parser.json

```diff
@@ -9,12 +9,13 @@
     "product_name": "Parser",
     "product_version_regex": ".*",
     "publisher": "Splunk",
-    "license": "Copyright (c) 2017-2021 Splunk Inc.",
+    "fips_compliant": true,
```

Comment:
This is required to be set to pass the static test. Also, on the excel sheet, Parser is shown to be fips compliant on the excel sheet.

Reply:
After doing global search for `hashlib.md5`, I don't see any `hashlib.md5` usage here.
See [reference](https://github.com/splunk-soar-connectors/crowdstrikeoauth/pull/1/files#diff-1ab4c4ae3c672ae5db676e2a88f429da50316741c9e072fc1a9673e9132f0ca8R278)

Reply:
The one hashlib usage is usee is `file_hash = hashlib.sha1(part_payload).hexdigest()` at https://github.com/splunk-soar-connectors/parser/blob/b50c7475faf41daaf1a3ae3a1a447e59007e56e3/parser_email.py#L673

Reply:
I believe it's fine to keep sha1 here.
cc: @chunmanjimmyf , @shiyingtu